src/codeu/chat/client/commandline/Chat.java

```diff
@@ -66,6 +96,19 @@ public boolean handleCommand(String line) {
       return true;
     }
 
+    if ("version-check".equals(command)) {
+      System.out.println("Version number: " + context.Version());
+      return true;
+    }
+
+    if ("up-time".equals(command)) {
+      Time serverTime = context.getTime();
+      Time now = Time.now();
+      double difference = now.inMs() - serverTime.inMs();
```

Comment:
Looking at the implementation of inMs() the return type is long, why is the difference a double?